tests/runner/src/util.rs

```diff
@@ -12,112 +12,107 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::fmt::Display;
 use std::net::SocketAddr;
 use std::path::PathBuf;
 use std::time::Duration;
 
 use client::api::v1::column::Values;
 use client::api::v1::ColumnDataType;
+use common_base::BitVec;
 use tokio::io::AsyncWriteExt;
 use tokio::net::TcpSocket;
 use tokio::time;
 
 /// Check port every 0.1 second.
 const PORT_CHECK_INTERVAL: Duration = Duration::from_millis(100);
+const NULL_DATA_PLACEHOLDER: &str = "__NULL__";
```

Comment:
Could we use `NULL` as what SQL displays?

Reply:
> Could we use `NULL` as what SQL displays?

I think this isn't a critical problem, so I'd also approve this PR.

Reply:
Using `NULL` is better.